plugin/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPlugin.kt

```diff
@@ -73,7 +74,7 @@ private fun Project.configurePlatform() {
 
 private fun Project.defaultJavaDocOption(): JavadocJar? {
   return if (plugins.hasPlugin("org.jetbrains.dokka") || plugins.hasPlugin("org.jetbrains.dokka-android")) {
-    JavadocJar.Dokka(provider { findDokkaTask() })
+    JavadocJar.Dokka(findDokkaTask())
```

Comment:
Moved `provider` to `findDokkaTask` to make it clearer that the content is evaluated lazily